vue.config.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   pluginOptions: {
     electronBuilder: {
       // List native deps here if they don't work
-      externals: ['keytar'],
+      externals: ['keytar', 'israeli-bank-scrapers-core'],
```

Comment:
@Arielgordon123 

> interesting, so the problem only happens if the import from `israeli-bank-scrapers-core` is to the store

No, the problem is when calling to `israeli-bank-scrapers-core`, who is calling to `ws`, in the `renderer` process. So the important change is this change, and this is why I said:

> Needs to validate the release output runs in the user machine.